src/tools/CMakeLists.txt

```diff
@@ -31,6 +31,7 @@ add_subdirectory(lb)
 add_subdirectory(load)
 add_subdirectory(pcp)
 add_subdirectory(post)
+add_subdirectory(recvfuzz)
```

Comment:
Maybe we make this Windows only for now? (put next to `etw` below, and update stress.yml to do Windows only)